reddit_service_activity/__init__.py

```diff
@@ -82,11 +82,16 @@ def make_processor(app_config):  # pragma: nocover
 
         "redis": {
             "url": config.String,
+            "max_connections": config.Optional(config.Integer, default=100),
         },
     })
 
     metrics_client = make_metrics_client(app_config)
-    redis_pool = redis.ConnectionPool.from_url(cfg.redis.url)
+    redis_pool = redis.BlockingConnectionPool.from_url(
+        cfg.redis.url,
+        max_connections=cfg.redis.max_connections,
+        timeout=0.1,
```

Comment:
yes, i'd like it to be. i'm gonna add support for "milliseconds" to [config.Timespan](https://reddit.github.io/baseplate/baseplate/config.html#baseplate.config.Timespan) first